src/constants.ts

```diff
@@ -21,6 +21,12 @@ export const configFileName: string = 'staticwebapp.config.json';
 
 export const showActionsMsg: MessageItem = { title: localize('openActions', 'Open Actions in GitHub') };
 
+export const cloneProjectMsg: MessageItem = { title: localize('cloneProject', 'Clone project from GitHub') };
```

Comment:
nit: No need to export these as constants since they're only used in one file (unless the next PR uses them too?)